decoder/vaapidecoder_mpeg2.cpp

```diff
@@ -289,9 +289,6 @@ YamiStatus VaapiDecoderMPEG2::start(VideoConfigBuffer* buffer)
         m_parser->nextStartCode(m_stream.get(), next_code);
         DEBUG("start Next start_code %x", next_code);
         status = processConfigBuffer();
-        if (status == YAMI_DECODE_FORMAT_CHANGE) {
```

Comment:
I agree, it should be uniform across decoders.  DECODE_FORMAT_CHANGE shouldn't be returned from start().  The interface documention already eludes to this: https://github.com/01org/libyami/blob/apache/interface/VideoDecoderInterface.h#L34